cohttp-curl-async/src/cohttp_curl_async.ml

```diff
@@ -168,7 +168,7 @@ module Request = struct
           timeout
       in
       Cohttp_curl.Request.create ?timeout_ms ?headers method_ ~uri ~input
-        ~output ~on_response:(Ivar.fill response_ready)
+        ~output ~on_response:(Ivar.fill_exn response_ready)
```

Comment:
Does this need a minimum version bump for async?

Reply:
Yes, `async_kernel >= v0.17.0`. I've added it to this PR.